app/src/main/java/com/deividasstr/BaseFragmentModule.kt

```diff
@@ -0,0 +1,25 @@
+package com.deividasstr
+
+import com.deividasstr.base.ActivityScope
+import com.deividasstr.base.FragmentScope
+import com.deividasstr.base.SingleIn
+import com.squareup.anvil.annotations.ContributesTo
+import com.squareup.anvil.annotations.MergeSubcomponent
+import dagger.Module
+import dagger.Subcomponent
+
+@ContributesTo(ActivityScope::class)
+@Module(subcomponents = [FragmentProvidersSubcomponent::class])
+abstract class BaseFragmentModule
```

Comment:
Why `BaseFragmentModule` contributes to ActivityScope? I guess, only base fragment's subcomponent or subcomponent's builder/factory should be contributed.

Or am I mistaken and scoped things in `BaseFragmentModule` won't outlive BaseFragment?